app/src/main/java/org/fossasia/susi/ai/activities/MainActivity.java

```diff
@@ -1,6 +1,7 @@
 package org.fossasia.susi.ai.activities;
 
 import android.Manifest;
+import android.app.Activity;
```

Comment:
@Abhi2424shek Yeah, will fix this asap. But do you have any idea why the circleci is failing?